t/op/sub.t

```diff
@@ -426,6 +426,7 @@ eval '
    CORE::state sub b; sub d { sub b {} sub d }
```

Comment:
The commit message should refer to https://www.nntp.perl.org/group/perl.daily-build.reports/2023/04/msg292891.html  (linked from the top of the page you linked) rather than perl.develop-help.com, while I don't plan to remove the site or change the URL structure, I might get hit by a bus, or run out of money or whatever.